### PR TITLE
add intitial healthcheck docs

### PR DESCRIPTION
**Pull request by dcmcand**
## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

**Comment by viniciusdc** [comment]
Hi @dcmcand thanks for the PR, can you address the failing test and also for the page to show up in the final doc you also need to include it in the https://github.com/nebari-dev/nebari-docs/blob/main/docs/sidebars.js#L55

File: docs/docs/how-tos/setup-healthcheck.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Overview
```
[the remark this answers is not in the document]
Comment (kcpevey) [reply docs/docs/how-tos/setup-healthcheck.md:1]: I added the formatting here, but feel free to change the title/description wording here.

**Comment by viniciusdc** [comment]
@dcmcand can you move this example to a code block? 
image

File: docs/docs/how-tos/setup-healthcheck.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+![Display of available kuberhealthy metrics in Grafana](/img/how-tos/nebari-healthchecks1.png)
+
+:::note
+
+If you have previously deployed Nebari without healthchecks, You may need to restart your Prometheus service to get it to pick up the kuberhealthy metrics.
+
+:::
+
+## Summary of available healthchecks
```
Comment (kcpevey) [inline docs/docs/how-tos/setup-healthcheck.md:52]: @dcmcand I added a table of available healthchecks to explain what each one does. Can you review and make sure this is correct?

Also, apologies for the unreadable format - yarn formatting added tons of spaces which makes it hard to read in md format.

**Review by kcpevey — APPROVED** [review]

**Pull request merged**